tests/models/test_transformers.py

```diff
@@ -57,6 +57,7 @@ def test_transformer_smoke_select(model_name, model_kwargs):
     assert lm["answer"] in ["p", "t", "w"]
 
 
+@pytest.mark.skip("Don't overload the build machines")
 def test_phi3_loading():
```

Comment:
LGTM for now, but perhaps we can configure these to run on larger spec'd machines after this PR?

Reply:
If you can provision them....